tests/testthat/test-stat_stepribbon.R

```diff
@@ -0,0 +1,130 @@
+#' @title Specifications test-stat_stepribbon.R
+#' @section Last updated by: Daniel Sjoberg (danield.sjoberg@@gmail.com)
+#' @section Last update date: 2022-07-12T04:47:32
+#'
+#' @section List of tested specifications
+#' T1. visR::stat_stepribbon() accepts aes() args ymin and ymax.
+#' T1.1 visR::stat_stepribbon() inherits aes from ggplot()
+#' T1.2 visR::stat_stepribbon() respects aes from visR::stat_stepribbon()
+#' T2. visR::stat_stepribbon() works with the color/fill aesthetic.
+#' T2.1 The step ribbon is correctly applied when there are multiple lines from the color/fill aesthetic
+#' T3. visR::stat_stepribbon(direction=) argument is respected.
+#' T3.1 The visR::stat_stepribbon(direction=) argument is able to produce figures based on both argument options.
+
+# Requirement T1 ----------------------------------------------------------
```

Comment:
could we make an attempt to decompose the final object to see if what is applied is what we expect?

Reply:
the ribbon difference updates how the interpolation between points is done, and I don't know how to check this inside the ggplot object, which is why i opted for the image snapshot.